cmd/cloud-endpoints-controller/main.go

```diff
@@ -90,6 +94,7 @@ func webhookHandler() func(w http.ResponseWriter, r *http.Request) {
 func sync(parent *CloudEndpoint, children *CloudEndpointControllerRequestChildren) (*CloudEndpointControllerStatus, *[]interface{}, error) {
 	status := makeStatus(parent, children)
 	currState := status.StateCurrent
+	log.Printf("[INFO] sync, %s - %s \n", parent.Name, currState)
```

Comment:
Remove this, it makes the controller logs very noisy as it syncs every 2 seconds.